webpack/webpack.common.js

```diff
@@ -54,9 +54,10 @@ module.exports = {
   plugins: [
     new webpack.ProvidePlugin({
       Buffer: ['buffer', 'Buffer'],
-    }),
-    new webpack.ProvidePlugin({
-      process: 'process/browser',
+
+      // We need to use require.resolve, otherwise we get this issue with
+      // react-redux: https://github.com/reduxjs/redux-toolkit/issues/4990
+      process: require.resolve('process/browser'),
```

Comment:
I got a build error without that..

I think we need `process` initially because of taskcluster-client-web otherwise.